### PR TITLE
Clean up tools and diagnostic messages

### PR DESCRIPTION
**Pull request by FUDCo**
Checking in some miscellaneous fixes to tooling and diagnostics which have accumulated, but would be nice to have merged before going on to the Next Thing.

### CODE REVIEW

**Review by warner — APPROVED** [review]
Looks good. In the long run we need something slog-like to coallate the comms debug messages with all the rest, so we can build a viewer application and do less manual printf/editor -based analysis, but this is a great start.

File: packages/swingset-runner/bin/runner-debug [hunk 1]
```diff
@@ -4,7 +4,11 @@
  * Simple boilerplate program providing linkage to launch an application written using modules within the
  * as yet not-entirely-ESM-supporting version of NodeJS.
  */
-import lmdb from 'node-lmdb';
+
+// LMDB bindings need to be imported before lockdown.
```
Comment (dckc) [inline packages/swingset-runner/bin/runner-debug:8]: why do LMDB bindings need to be imported before lockdown?

File: packages/SwingSet/src/vats/comms/cdebug.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+let enableCDebug = false;
```
Comment (dckc) [inline packages/SwingSet/src/vats/comms/cdebug.js:1]: global mutable state. hm.

Adding this to the list of things to think about in #2160

Comment (FUDCo) [reply packages/SwingSet/src/vats/comms/cdebug.js:1]: It's not global.

File: packages/swingset-runner/bin/runner-debug (hunk 1 above)
Comment (FUDCo) [reply packages/swingset-runner/bin/runner-debug:8]: Weird native code stuff.

File: packages/SwingSet/src/vats/comms/cdebug.js (hunk 2 above)
Comment (dckc) [reply packages/SwingSet/src/vats/comms/cdebug.js:1]: In what way is it not global? `cdebugEnable` is exported, so the flag can be changed from anywhere.

Reading the flag is a little more involved (stupid `toString` tricks) but it's not prevented.

**Pull request merged**